src/controller/CHIPDeviceController.cpp

```diff
@@ -47,6 +47,9 @@
 #include <app/util/DataModelHandler.h>
 #include <app/util/error-mapping.h>
 #include <credentials/CHIPCert.h>
+#include <credentials/DeviceAttestationCredsProvider.h>
+#include <credentials/examples/DeviceAttestationCredsExample.h>
+#include <credentials/examples/DeviceAttestationVerifierExample.h>
```

Comment:
The example headers don't belong here. They only belong in the applications that instantiate that instantiate `DeviceCommissioner`, and call `SetDeviceAttestationVerifier`. Please remove from here.